topicgenerator/results/generated-topics_CLAUDE_3_HAIKU_20241003T172935Z.csv

```diff
@@ -0,0 +1,24 @@
+trsId,version,descriptorUrl,descriptorChecksum,isTruncated,promptTokens,completionTokens,cost,finishReason,aiTopic
+"#workflow/github.com/iwc-workflows/sars-cov-2-pe-illumina-artic-variant-calling/COVID-19-PE-ARTIC-ILLUMINA",main,https://raw.githubusercontent.com/iwc-workflows/sars-cov-2-pe-illumina-artic-variant-calling/main//pe-artic-variation.ga,dcc2761eb35156d7d09479112daf089439774fc29938f02cb6ee8cda87906758,false,24598,54,0.0062169999999999994,end_turn,"Processes Illumina-sequenced ARTIC data by mapping and variant calling, trimming ARTIC primer sequences, identifying primer-binding site mutations, and excluding reads from affected amplicons to calculate allele-frequencies."

```

Comment:
Stylistically, CLAUDE seems more prone to generate long compound sentences, the differences between the models are interesting...